GiftedListView.js

```diff
@@ -200,6 +202,16 @@ var GiftedListView = React.createClass({
     this.props.onFetch(this._getPage(), this._postRefresh, {firstLoad: true});
   },
 
+  componentWillReceiveProps() {
+    if (this.props.forceUpdate) {
```

Comment:
And the change this to `nextProps.forceUpdate` otherwise you need two cycles before it updates